website/tests/test_bidding.py

```
@@ -26,7 +26,7 @@ def setUp(self):
     def test_add_bid_authenticated_user(self):
         """Test adding a bid with an authenticated user."""
         # Log in the user
-        self.client.login(username="testuser", password="testpassword")
+        self.client.login(email="test@example.com", password="testpassword")
```

Ruff (0.14.7)

Possible hardcoded password assigned to argument: "password"

(S106)